chain/rosetta-rpc/src/models.rs

```diff
@@ -53,13 +61,20 @@ pub(crate) struct AccountIdentifier {
      * /// Blockchains that utilize a username model (where the address is not a
      * /// derivative of a cryptographic public key) should specify the public
      * /// key(s) owned by the address in metadata.
-     * #[serde(skip_serializing_if = "Option::is_none")]
-     * pub metadata: Option<serde_json::Value>, */
+     */
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub metadata: Option<Vec<PublicKey>>,
+}
+
+impl PartialEq for AccountIdentifier {
+    fn eq(&self, other: &Self) -> bool {
+        self.address == other.address
```

Comment:
Feels odd that we don't compare subacocunts here. I don't unserstand the intended semantics, but I thing we perhaps should compare subaccounts?